sdk/src/Services/S3/Custom/Transfer/TransferUtilityGetObjectResponseBase.cs

```diff
@@ -0,0 +1,293 @@
+/*******************************************************************************
```

Comment:
made this base class because stream and file based downloads have the same properties. the only difference is openstream command will have extra response stream field